components/org.wso2.carbon.identity.api.server.webhook.management/org.wso2.carbon.identity.api.server.webhook.management.v1/src/main/java/org/wso2/carbon/identity/api/server/webhook/management/v1/core/ServerWebhookManagementService.java

```diff
@@ -145,12 +145,13 @@ public void deleteWebhook(String webhookId) {
      * Activate webhook.
      *
      * @param webhookId Webhook ID.
+     * @return Activated webhook.
      */
-    public void activateWebhook(String webhookId) {
+    public WebhookResponse activateWebhook(String webhookId) {
 
         try {
-            webhookManagementService.activateWebhook(webhookId,
-                    CarbonContext.getThreadLocalCarbonContext().getTenantDomain());
+            return getWebhookResponse(webhookManagementService.activateWebhook(webhookId,
```

Comment:
This webhook response should be similar to the response you return on list. This means all channel related info is not needed.

Reply:
fixed with https://github.com/wso2/identity-api-server/pull/905/commits/6037694a0c9c3fd6cecef37039b5ded27025fdaf